cmd/create/oidcconfig/cmd.go

```diff
@@ -110,23 +120,86 @@ func run(cmd *cobra.Command, argv []string) {
 			r.Reporter.Errorf("Expected a valid OIDC provider creation mode: %s", err)
 			os.Exit(1)
 		}
+
+		kmsArn, err = interactive.GetString(
+			interactive.Input{
+				Question: "KMS ARN",
+				Help:     cmd.Flags().Lookup("kms-arn").Usage,
+				Required: true,
+				Default:  args.kmsArn,
+			})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid KMS ARN: %s", err)
+			os.Exit(1)
+		}
+	}
+
+	if kmsArn != "" {
+		_, err := arn.Parse(kmsArn)
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid policy ARN for permissions boundary: %s", err)
```

Comment:
Should the error message be related to the `kmsARN`?

Reply:
Thanks! fixed